src/assisted_installer_controller/assisted_installer_controller_test.go

```diff
@@ -553,6 +553,36 @@ var _ = Describe("installer HostRoleMaster role", func() {
 			mockbmclient.EXPECT().CompleteInstallation(gomock.Any(), "cluster-id", true, "").Return(fmt.Errorf("dummy")).Times(1)
 			mockbmclient.EXPECT().CompleteInstallation(gomock.Any(), "cluster-id", true, "").Return(nil).Times(1)
 
+			wg.Add(1)
+			assistedController.PostInstallConfigs(&wg, status)
+			wg.Wait()
+			Expect(status.HasError()).Should(Equal(false))
+		})
+		It("Run PostInstallConfigs - waiting for single OLM operator which timeouts", func() {
+			cmName := "default-ingress-cert"
+			cmNamespace := "openshift-config-managed"
+			consoleOperatorName := "console"
+			data := make(map[string]string)
+			data["ca-bundle.crt"] = "CA"
+			cm := v1.ConfigMap{Data: data}
+			finalizing := models.ClusterStatusFinalizing
+			cluster := models.Cluster{Status: &finalizing}
+			WaitTimeout = 5 * time.Second
+			mockbmclient.EXPECT().GetCluster(gomock.Any()).Return(&cluster, nil).Times(1)
+			mockk8sclient.EXPECT().GetConfigMap(cmNamespace, cmName).Return(&cm, nil).Times(1)
+			mockbmclient.EXPECT().UploadIngressCa(gomock.Any(), data["ca-bundle.crt"], assistedController.ClusterID).Return(nil).Times(1)
```

Comment:
It would be nice if you could move all of this section into a helper function `SetControllerWaitForOLMOperators` or something like that